content/docs/concepts/low-power.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Low Power"
+linkTitle: "Low Power"
+type: "docs"
+weight: 35
+description: >
+  How to consume less power on microcontrollers
+---
+
+Microcontrollers are usually rather non-demanding when it comes to power consumption.
+Couple of watts is the most you may expect a hobby project would use.
+
+Sometimes, that is not low enough.
+Power your project from a battery, and it'd benefit greatly from consuming less.
+
+Common scenario for such projects is sleeping most of the time, and only rarely briefly wake up to query sensors and execute actions.
+The main caveat here is if not configured properly, microcontroller and periferals consume power as usual while your program "sleeps".
+
+To address that, most microcontrollers have one or more "low power" modes, also known as "deep sleep" and, sometimes, "dormant".  
+In such modes microcontroller may consume as low as milli- or even micro-Watts.  
```

Comment:
Maybe this would be a good place to make the distinction between `time.Sleep` and these "deeper" sleeps? i.e:

When writing normal Go code one has the standard library `time.Sleep` function at disposal. While this sleep saves us power and is more efficient than looping until a certain time has passed, there are other ways of letting the CPU sleep. When one has baremetal control of the hardware, as is the case when writing TinyGo for flashing to microcontroller chips, it is possible to put several critical parts of a micocontroller to sleep, and sometimes put the uC to sleep entirely to save power. 

Text is for demo. has not been reviewed closely

Reply:
Thank you, @soypat, changes applied